server/src/main/java/io/druid/segment/realtime/appenderator/StreamAppenderatorDriver.java

```diff
@@ -54,11 +53,11 @@
 
 /**
  * This class is specialized for streaming ingestion. In streaming ingestion, the segment lifecycle is like:
- *
+ * <p>
```

Comment:
I think we avoid these `<p>` tags since the linter doesn't like them. Try turning off "Generate p on empty lines" in the JavaDoc section of your code style pane.

Reply:
fixing